### ISSUE TITLE
Use `line-height: 1` as the default for the `ButtonBase` component

### ISSUE THREAD

**Issue by cheton** [trendmicro-frontend/tonic-ui#123]
![image](https://user-images.githubusercontent.com/447801/80776356-c8e39480-8b94-11ea-934a-456443386b6f.png)

As a base component, I suggest that we set its `line-height` to `1` in the `ButtonBase` component rather than inheriting `button` styles from browser default or CSS normalize. This can make sure the `ButtonBase` has an exact height as the children when implementing an `IconButton` like below:

```jsx
<ButtonBase>
    <Icon name="icon-name" /> /* 16px x 16px */
</ButtonBase>
```

**Comment by cheton** [comment]
Also set `line-height: 1` for TMIcon as well.

**Comment by cheton** [comment]
Fixed in #125
